spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -124,10 +128,20 @@
     normalized_params[:bcc].should eq []
   end
 
+  it 'distinguishes between angle brackets in the display name and the email address' do
+    params = default_params.merge(
+      to: '"<Display name>" <email@example.com>',
+      envelope: '{"to":["email@example.com"]}',
+    )
+    normalized_params = normalize_params(params)
+
+    normalized_params[:bcc].should eq []
```

Comment:
Shouldn't this assert `normalized_params[:to].should eq ['"<Display name>" <email@example.com>']`?